docs/config.md

```diff
@@ -0,0 +1,25 @@
+How to configure search bar to be able to search on more categories.
+Directions:
+*First decide what you what to search on e.g Facility Name
+*Then go find the appropriate table in the DB that contains the column you want to search on
+*Then go into your docker-compose folder and open the search.json file
```

Comment:
Rather keep this as generic as possible and rather say that they need to edit the `search.json` file in the config repo. As in alot of cases the person making these config changes will not even be running the code on their own machine.